### PR TITLE
API: delegation | delegation update

### PR DESCRIPTION
**Pull request by Sotatek-DavidNguyen4**

### CODE REVIEW

**Comment by Sotatek-DavidNguyen4** [comment]
image

File: src/main/java/microservices/delegation/steps/DelegationControllerSteps.java [hunk 1]
```diff
@@ -56,4 +56,16 @@ public DelegationControllerSteps verifyFormatAttributes(PoolDetailDelegatorModel
...
         return this;
     }
+    @Step("verify format attributes")
+    public DelegationControllerSteps verifyFormatAttributes(PoolDetailHeaderModel actualPoolDetailHeader){
+        Assert.assertTrue(AttributeStandard.isValidDateFormat(actualPoolDetailHeader.getCreateDate(), DATE_FORMAT[0]));
+        return this;
+    }
+    @Step("verify format attributes")
+    public DelegationControllerSteps verifyHashViewFormat(PoolDetailHeaderModel actualPoolDetailHeader){
+        System.out.println(actualPoolDetailHeader.getHashView());
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/delegation/steps/DelegationControllerSteps.java]: remove it

File: src/main/java/microservices/delegation/steps/DelegationHeaderSteps.java [hunk 2]
```diff
@@ -11,6 +11,7 @@ public class DelegationHeaderSteps extends BaseApi {
     @Step("get data for delegation headers")
     public DelegationHeaderSteps getDataForDelegationHeader(){
         sendGet(Endpoints.DelegationApi.POOL_HEADER_URI);
+        System.out.println(getResponse().getBody().print());
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/delegation/steps/DelegationHeaderSteps.java]: remove it

File: src/test/java/tests/delegation_controller/DelegationEpochListTests.java [hunk 3]
```diff
@@ -15,66 +16,81 @@
...
         param = new CreateParameters()
                 .withPoolView(poolView)
-                .withPageSize(size)
+                .withPage(page)
                 .getParamsMap();
 
         poolDetailEpochs = (PoolDetailEpochs) delegationDetailEpchoSteps
                 .getDataForPoolDetailEpochs(param)
-                .validateStatusCode(HttpURLConnection.HTTP_OK)
+                .validateStatusCode(HttpURLConnection.HTTP_BAD_REQUEST)
```
Comment (Sotatek-DavidNguyen4) [inline src/test/java/tests/delegation_controller/DelegationEpochListTests.java:59]: Does miss validate error message and error code?

Comment (Sotatek-DavidNguyen4) [reply src/test/java/tests/delegation_controller/DelegationEpochListTests.java:59]: because this tcs just require like that

**Review by ghost — APPROVED** [review]

**Pull request merged**